core-bundle/src/HttpKernel/ModelArgumentResolver.php

```diff
@@ -44,11 +44,7 @@ public function supports(Request $request, ArgumentMetadata $argument): bool
             return false;
         }
 
-        if (!$argument->isNullable() && null === $this->fetchModel($request, $argument)) {
-            return false;
-        }
-
-        return true;
+        return !(!$argument->isNullable() && null === $this->fetchModel($request, $argument));
```

Comment:
@ausi Would `return $argument->isNullable() || null !== $this->fetchModel($request, $argument);` be the same?

Reply:
Yes.

Reply:
Also reverted in f8fc72b51bd70224eb51a04d462fc2e3ce04206b.